anxcloud/provider.go

```diff
@@ -32,14 +32,16 @@ func Provider() *schema.Provider {
 			},
 		},
 		ResourcesMap: map[string]*schema.Resource{
-			"anxcloud_virtual_server":     resourceVirtualServer(),
-			"anxcloud_vlan":               resourceVLAN(),
-			"anxcloud_network_prefix":     resourceNetworkPrefix(),
-			"anxcloud_ip_address":         resourceIPAddress(),
-			"anxcloud_tag":                resourceTag(),
-			"anxcloud_dns_zone":           resourceDNSZone(),
-			"anxcloud_dns_record":         resourceDNSRecord(),
-			"anxcloud_lbaas_loadbalancer": resourceLBaaSLoadBalancer(),
+			"anxcloud_virtual_server":        resourceVirtualServer(),
+			"anxcloud_vlan":                  resourceVLAN(),
+			"anxcloud_network_prefix":        resourceNetworkPrefix(),
+			"anxcloud_ip_address":            resourceIPAddress(),
+			"anxcloud_tag":                   resourceTag(),
+			"anxcloud_dns_zone":              resourceDNSZone(),
+			"anxcloud_dns_record":            resourceDNSRecord(),
+			"anxcloud_lbaas_loadbalancer":    resourceLBaaSLoadBalancer(),
+			"anxcloud_kubernetes_cluster":    resourceKubernetesCluster(),
```

Comment:
We've previously talked about naming Kubernetes TF resources and decided to use `anxcloud_k8s_cluster` and `anxcloud_kubeconfig`. I'm not sure about that decision and think it would be better if it's namespaced as implemented right now. Could we maybe reconsider the names?

Reply:
* anxcloud_kubernetes_cluster: 
* anxcloud_kubernetes_kubeconfig: , waiting for @toothstone response on discussion below